client/my-sites/media/main.jsx

```diff
@@ -238,7 +238,7 @@ export default React.createClass( {
 					<MediaLibrarySelectedData siteId={ site.ID }>
 						<MediaLibrary
 							{ ...this.props }
-							className="media__main-section"
+							className="media__primary"
```

Comment:
I had to add a child wrapper since there is a needed reference used to compute the width of the library.
<Main /> is a stateless component, therefore, It doesn't support refs.